deploy/import-upstream.sh

```diff
@@ -1,2 +1,14 @@
-oc import-image coreos/prometheus-operator:v0.25.0 --from=quay.io/coreos/prometheus-operator:v0.25.0 --confirm
-oc import-image coreos/configmap-reload:v0.0.1 --from=quay.io/coreos/configmap-reload:v0.0.1 --confirm
+#!/bin/sh
+oc import-image prometheus:v2.11.104 --from=quay.io/prometheus/prometheus --confirm
+oc import-image prometheus-operator:latest --from=quay.io/coreos/prometheus-operator:v0.27.0 --confirm
+oc import-image prometheus-configmap-reloader:latest --from=quay.io/coreos/configmap-reload --confirm
+oc import-image prometheus-config-reloader:latest --from=quay.io/coreos/prometheus-config-reloader:v0.30.1 --confirm
+oc import-image prometheus-alertmanager:v0.15.0 --from=quay.io/prometheus/alertmanager --confirm
+
+oc import-image amq-interconnect:1.4-6 --from=quay.io/interconnectedcloud/qdrouterd:1.7.0 --confirm
```

Comment:
```
<leifmadsen> yea... what we really want is 1.7.0...
<leifmadsen> that's what we have downstream
<leifmadsen> so yea... the version scheme frankly sucks
<leifmadsen> 1.3 == 1.5, 1.4 == 1.7
<leifmadsen> amq-interconnect vs qdrouterd
<csibbitt> Oh yeah? haha. Okay! That I can handle. If the downstream amq-interconnect:1.4  is equivalent to upstream qdrouterd:1.7.0  (I'll just take your word for it unless you have an easy reference I can read?) - then we're in good shape.
<leifmadsen> I just kind of know from memory
```

Reply:
Follow up on this based on version discussions we're having. If we're okay with trying to use "latest" for internal tags for now because the source tag is more specific anyways, should we use `:latest` here, too? IIUC, the prometheus operator demands :vX.Y.Z versions for it's containers, but it seems like we could use 'latest' for everything else. 

I'm still not 100% comfortable with "latest" because it implies a certain behavior that you might not actually get (for example if you override the imagePullPolicy to something other than Always, or whether you do or do not have an Image Change Trigger in your deployment). I think this is part of why some people use a "master" tag. I'm okay with keeping it for now because I'm not familiar enough with ImageStreams to make a solid recommendation today, but if we can then we should at least try to be 100% consistent.

Reply:
Follow up on this based on version discussions we're having. If we're okay with trying to use "latest" for internal tags for now because the source tag is more specific anyways, should we use :latest here, too? IIUC, the prometheus operator demands :vX.Y.Z versions for it's containers, but it seems like we could use 'latest' for everything else.

I'm still not 100% comfortable with "latest" because it implies a certain behavior that you might not actually get (for example if you override the imagePullPolicy to something other than Always, or whether you do or do not have an Image Change Trigger in your deployment). I think this is part of why some people use a "master" tag. I'm okay with keeping it for now because I'm not familiar enough with ImageStreams to make a solid recommendation today, but if we can then we should at least try to be 100% consistent.

Reply:
Notes have been added to help clarify why we're doing things the way we're doing it. Additional work is likely required here, but I'm hoping that our eventual migration to OLM will result in a lot of this stuff just taking care of itself.